README.md

```diff
@@ -40,6 +40,13 @@ You can also use [Glide](https://github.com/Masterminds/glide) to manage depende
 ```  
 glide install  
 ```  
+
+## Configuration  
```

Comment:
This should be Credential Configuration, since X-Ray daemon can be configured through cmd or cfg.yaml file.